src/routes/public/policies.js

```diff
@@ -26,7 +29,11 @@ exports.register = function (server, options, next) {
       validate: {
         headers: Joi.object({
           'authorization': Joi.any().required()
-        }).unknown()
+        }).unknown(),
+        query: Joi.object({
+          page: Joi.number().integer().positive().required().description('Page number, starts from 1'),
+          limit: Joi.number().integer().positive().required().description('Items per page')
```

Comment:
It mostly depends on how you see URLs. What I've seen in the project so far is that they are used as resource identifiers. Putting options like page & limit in the URLs will divert from that practice. My opinion is that these properly belong to query params.

Agree on the defaults. It make sense to have them when we return the page number and limit in a result wrapper.